src/main/scala/com/stratio/khermes/clients/shell/KhermesConsoleHelper.scala

```diff
@@ -192,12 +132,96 @@ case class KhermesConsoleHelper(client: KhermesClientActor) {
     reader.setExpandEvents(false)
-    reader.setPrompt("khermes> ")
+    val completer=new StringsCompleter("bye","clear","exit","help","ls","save","show","start","stop","quit")
+    reader.addCompleter(completer)
+    val argumentShowCompleter =new ArgumentCompleter(
+      new StringsCompleter("show"),
+      new StringsCompleter("--kafka","--template", "--khermes","--avro"),
+      new NullCompleter())
+    val argumentSaveCompleter =new ArgumentCompleter(
+      new StringsCompleter("save"),
+      new StringsCompleter("--kafka", "--template", "--khermes","--avro"),
+      new NullCompleter())
+    val argumentStartCompleter =new ArgumentCompleter(
+      new StringsCompleter("start"),
+      new StringsCompleter("--kafka","--template","--khermes","--avro","--ids"),
+      new NullCompleter())
+    val argumentStopCompleter =new ArgumentCompleter(new StringsCompleter("stop"), new StringsCompleter("--ids"), new NullCompleter())
+    reader.addCompleter(argumentShowCompleter)
+    reader.addCompleter(argumentSaveCompleter)
+    reader.addCompleter(argumentStartCompleter)
+    reader.addCompleter(argumentStopCompleter)
+    reader.setPrompt("\u001B[33mkhermes> \u001B[0m")
     reader
   }
 }
 
 object KhermesConsoleHelper {
 
   val TimeoutWhenLsMessage = 200L
-}
+
+  def printNotFoundCommand: String = {
+    "Command not found. Type help to list available commands."
+  }
+
+  def help: String = {
+    s"""Khermes client provides the next commands to manage your Khermes cluster:
+       |  Usage: COMMAND [args...]
+       |
+               |  Commands:
+       |     start [command options] : Starts event generation in N nodes.
+       |       --khermes    : Khermes configuration
+       |       --kafka      : Kafka configuration
+       |       --template   : Template to generate data
+       |       --avro       : Avro configuration
+       |       --ids        : Node id where start khermes
+       |     stop [command options] : Stop event generation in N nodes.
+       |       --ids        : Node id where start khermes
+       |     ls                    : List the nodes with their current status
+       |     save [command options] : Save your configuration in zookeeper
+       |       --khermes    : Khermes configuration
+       |       --kafka      : Kafka configuration
+       |       --template   : Template to generate data
+       |       --avro       : Avro configuration
+       |     show [command options] : Show your configuration
+       |       --khermes    : Khermes configuration
+       |       --kafka      : Kafka configuration
+       |       --template   : Template to generate data
+       |       --avro       : Avro configuration
+       |     clear                 : Clean the screen.
+       |     help                  : Print this usage.
+       |     exit | quit | bye     : Exit of Khermes Cli.   """.stripMargin
+  }
+
+  def show(args: Map[String, String], firstWord: String): String = {
+    Try {
```

Comment:
Take a walk on this portion of code because you can not use .get and .head. You are breaking the monad encapsulation.